media/css/plugincheck/plugincheck.less

```diff
@@ -381,10 +387,215 @@ html[dir="rtl"] {
+
+        @media only screen and (max-width: @breakDesktop) {
+            width: 95px;
+        }
+
+        @media only screen and (max-width: @breakTablet) {
+            display: block;
+            margin: 0 auto;
+            width: 95px;
+        }
+    }
+
+    #main-feature {
+        margin: -80px 0 0 165px;
+
+        @media only screen and (max-width: @breakDesktop) {
+            margin: -80px 0 0 115px;
+        }
+
+        @media only screen and (max-width: @breakTablet) {
+            margin: -20px 0 0;
+            text-align: center;
+        }
+    }
+}
+
+.newsletter-container-variant {
+    background-color: #00a5e1;
+    color: #fff;
+    margin-bottom: 30px;
+    padding: @baseLine * 2;
+    border-radius: 0;
+
+    &:before {
+        display: inline-block;
+        content: '';
+        .at2x('/media/img/plugincheck/newsletter.png', 149px, 149px);
+        background-repeat: no-repeat;
+        padding: 10px;
+        width: 149px;
+        height: 149px;
+
+        @media only screen and (max-width: @breakDesktop) {
+            display: block;
+            margin: 0 auto;
+        }
+
+        @media only screen and (max-width: @breakTablet) {
+            display: none;
+        }
+    }
+
+    header,
+    #newsletter-form {
+        display: inline-block;
+        margin-top: 18px;
+        padding: 0;
+        width: 350px;
+        vertical-align: top;
+
+        @media only screen and (max-width: @breakDesktop) {
+            display: block;
+            margin: 0 auto;
+        }
+
+        @media only screen and (max-width: @breakMobileLandscape) {
+            width: 240px;
+        }
+    }
+
+    #newsletter-form {
+        .form-contents,
+        .form-submit {
+            display: block;
+            width: 100%;
+        }
+
+        a {
+            color: #fff;
+
+            &:hover,
+            &:focus,
+            &:active {
+                color: #fff;
+            }
+        }
+
+        input[type="email"] {
+            width: 94%;
+            border-radius: 3px;
+
+            @media only screen and (max-width: @breakMobileLandscape) {
+                width: 90%;
+            }
+        }
+
+        input[type="submit"] {
+            background-color: transparent;
+            border: 2px solid #fff;
+            border-radius: 3px;
+
+            &:hover,
+            &:focus {
+                background-color: transparent;
+            }
+        }
+    }
+
+    h3,
+    h4 {
+        color: #fff;
+        text-shadow: none;
+
+        @media only screen and (max-width: @breakDesktop) {
+            text-align: center;
+        }
+    }
+
+    h3 {
+        .font-size(3.5rem);
+        line-height: .7;
+
+        @media only screen and (max-width: @breakMobileLandscape) {
+            .font-size(2.5rem);
+        }
+
+        .firefox {
+            font-weight: bold;
+        }
+    }
+
+    h4 {
+        .font-size(2rem);
+        line-height: 1.2;
+        letter-spacing: 0;
```

Comment:
I think this should be `letter-spacing: normal;` though the effect looks to be the same (probably depends on fonts).